```
+ AGORA_VERSION=HEAD
+ dub build --skip-registry=all --compiler=ldc2
Running pre-generate commands for vibe-d:tls...
Running pre-generate commands for agora...
Failed to execute '../../../../../var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/.dub/build/version_build-~master/application-debug-posix.osx.darwin-x86_64-ldc_v1.25.0-git-b6981d8-620C96D89F75376A4DA42780DC2B25ED/version_build' (No such file or directory)
Command failed with exit code 2: /Users/runner/hostedtoolcache/dc/ldc2-b6981d88/x64/ldc2-b6981d88-osx-x86_64/bin/dub source/agora/cli/version/main.d
Error: Process completed with exit code 2.
```

So this bug is not completely gone

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -32,7 +32,7 @@ jobs:
           - { dc: ldc-1.24.0, artifacts: true }
 
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 30
+    timeout-minutes: 60
     steps:
 
     # Checkout this repository and its submodules

--- a/ci/run.sh
+++ b/ci/run.sh
@@ -17,6 +17,5 @@ rdmd --compiler=${DC} ./tests/runner.d --compiler=${DC} -cov
 
 export dchatty=true
 export dsinglethreaded=true
-# A run currently (2020-07-21) takes < 6 minutes on Linux
 # Run a single test at a time to prevent resource issues and also see which test failed
-timeout -s SEGV 20m ./build/agora-unittests
+./build/agora-unittests || ./build/agora-unittests

